boilerplate/csswg/status.include

```diff
@@ -75,7 +75,7 @@
 	as a <strong>First Public Working Draft</strong> using the <a
       href='https://www.w3.org/2021/Process-20211102/#recs-and-notes'>Recommendation
       track</a>.
-	Publication as a Working Draft
+	Publication as a First Public Working Draft
```

Comment:
This isn't wrong, but I don't know if the increased specificity is useful either. That statement is true of any working draft, including the first one, but not limited to it. If pubrules insists on this, then sure, whatever. But why does pubrules insist on this kind of irrelevant rephrasing?

Reply:
I don't know, I just want to get stuff published with the minimum amount of hacking on generated HTML. Which then has to be repeated each publication. Or indeed multiple times if the .bs  also has to be edited, for example to fix broken links.

Reply:
> But why does pubrules insist on this kind of irrelevant rephrasing?

I suspect, in this case, because it makes it easier to do the string matching to check specific sentences are included.

The impact on tooling for these constant changes is not, it seems, considered at all.

Reply:
@frivoal you aren't asking for changes, so please review and merge? I'm publishing another FPWD today and once again making this edit manually on the generated HTML.

Reply:
I cannot merge, only @tabatkins can do that. But I'm happy to mark this as accepted if that helps get there.